runtime/basilisk/src/lib.rs

```diff
@@ -171,12 +172,9 @@ construct_runtime!(
 		Utility: pallet_utility = 5,
 		//NOTE: 6 - is used by Scheduler which must be after cumulus_pallet_parachain_system
 		Democracy: pallet_democracy exclude_parts { Config } = 7,
-		Elections: pallet_elections_phragmen = 8,
```

Comment:
We should check the storage after we remove pallets, I think we'll need to clear it either by ref or by migration.

Reply:
Ok, I see later we do migration